src/models/PMCMean.h

```diff
@@ -0,0 +1,37 @@
+#ifndef PMCMEAN
+#define PMCMEAN
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
`stdlib.h` er deprecated i C++, brug `cstdlib` i stedet for.

Reply:
fixed